install/kubernetes/helm/istio/values.yaml

```diff
@@ -37,7 +40,9 @@ global:
     #     Redirect only selected ports:            --includeInboundPorts="80,8080"
     includeInboundPorts: "*"
     excludeInboundPorts: ""
-    policy: enabled
+
+    # This controls the 'policy' in the sidecar injector.
+    autoinject: enabled
```

Comment:
should this be autoInject for consistency?

Reply:
good point. Policy was very confusing - we also have the mixer policy.